### PR TITLE
Admin Template

### PR DESCRIPTION
**Pull request by vietfelix**

### CODE REVIEW

**Comment by vietfelix** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -11,6 +11,8 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
+/*---------------------- Admin Route --------------------*/
+Route::group(['prefix' => 'admin'], function() {
+
 });
+/*-------------------------------------------------------*/
```
Comment (BunnyPi04) [inline routes/web.php]: thiếu dòng trắng cuối file
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_23-dong-trong-8

**Comment by BunnyPi04** [comment]
commented

**Comment by vietfelix** [comment]
ready

**Pull request merged**